sci-astronomy/wcslib/wcslib-7.11.ebuild

```diff
@@ -60,8 +56,7 @@ src_configure() {
 }
 
 src_test() {
-	# -j1 to work around a race condition
-	emake -j1 check
+	emake check
```

Comment:
I'll fix on merge, but we can just drop src_test entirely then as the default definition will do this. Thanks!